### PR TITLE
add yaqc-qtypes

### PR DESCRIPTION
**Pull request by untzag**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by untzag** [comment]
@ksunden please comment if willing

**Comment by ksunden** [comment]
yes, willing

**Comment by untzag** [comment]
Don't understand why pip check fails, will return later to think about it.

**Comment by ksunden** [comment]
pip check failure should be solved by conda-forge/PySide2-feedstock#92, I think we can remove the check for now and move forward. The problem was that PySide2 _was_ installed, just without the distinfo so pip was unaware that it was installed, so pip check failed as it is a dependency.

File: recipes/yaqc-qtpy/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+    - qtpy
+    - qtypes >=0.3.0
+    - yaqc
+    - yaqd-core >=2020.06.3
+
+test:
+  imports:
+    - yaqc_qtpy
+  commands:
+    - pip check
```
Comment (ksunden) [inline recipes/yaqc-qtpy/meta.yaml:40]:
```suggestion
```

File: recipes/yaqc-qtpy/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+    - pip
+    - python >=3.7
+    - flit
+    - yaqc
+    - qtpy
+    - pyside2
+  run:
+    - pyside2
+    - python >=3.7
+    - qtpy
```
Comment (ksunden) [inline recipes/yaqc-qtpy/meta.yaml:31]:
```suggestion
    - qtpy
    - click
```

**Comment by ksunden** [comment]
(PR submitted upstream to add click dep, click is required for the cli app, though not raw import... sooo I suppose it could still be left out here, but just added for the test)

**Comment by untzag** [comment]
I'm giving up, will open a separate PR after releasing upstream and thinking more

**Pull request closed without merging**